packages/components/src/AppGuidedTour/DemoContentStep.component.js

```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+import Icon from '../Icon';
+import Stepper from '../Stepper';
+import { getI18nInstance } from '../translate';
+
+import theme from './DemoContentStep.scss';
+
+const i18n = getI18nInstance();
+
+export default function DemoContentStep({ demoContentSteps }) {
+	if (!demoContentSteps.length) {
+		return null;
+	}
+
+	return (
+		<React.Fragment>
+			<p className={theme.info}>
+				{i18n.t('tui-components:DEMO_CONTENT_LOADING_MESSAGE', {
```

Comment:
I'm not sure about the `tui-components` namespace, don't think you need to declare it?

Reply:
tui-components is the namespace to use for the components. How can i declare it ? The extract is correct for the component

Reply:
Do we tend to use useTranslation with the namespace in components rather than using the i18n shared instance ?
I think the main difference would be to be connected to the react lifecycle, so this implementation will work of course, it's more a manner of what we want to do.
In general, i'm using useTtranslation in react components and global i18n instance when i can't use the hook (as used in saga or reducer for instance)

Do we have some guidelines about this ? cc @jsomsanith-tlnd

Reply:
Ok good, thx for the precision anyway

Reply:
I replaced by useTranslation, it's better indeed